### PR TITLE
Make cram tests private

### PR DESCRIPTION
**Pull request by pitag-ha**

### CODE REVIEW

File: test/driver/error_embedding/test.t/run.t [hunk 1]
```diff
@@ -7,13 +7,13 @@ With the `-embed-errors` options, if a PPX raises, the first such exception
 is caught and packed into an output AST
 
   $ echo "let _ = [%raise]" > impl.ml
-  $ raiser -embed-errors impl.ml
+  $ ./../raiser.exe -embed-errors impl.ml
```
Comment (NathanReb) [inline test/driver/error_embedding/test.t/run.t:10]: Isn't `../raiser.exe` enough here? Same for the other occurences in this file and some of the others!

**Review by NathanReb — COMMENTED** [review]
Looks good, just one minor comment!

File: test/driver/error_embedding/test.t/run.t (hunk 1 above)
Comment (pitag-ha) [reply test/driver/error_embedding/test.t/run.t]: Good catch, thanks!

**Review by NathanReb — APPROVED** [review]
Thanks!

Can you add a changelog entry mentioning we fixed a bug where we would install unwanted public binaries?

**Review by NathanReb — APPROVED** [review]

**Pull request merged**